<launch>

    <!-- Group for imageProcessing-->
    <group ns="imageProcessing">

        <!--Including launch file for external camera imageProcessing -->
        <include file="$(find ibo1_irc_api)/launch/ec_imageProcessing.launch">
            <arg name="input_max_corners" value="100"/>
            <arg name="debug" value="${arg debug}"/>
        </include>
    </group>

    <!-- <group ns="my_gen3">
        Starting the CreateTarget_node
	    <node pkg="ibo1_irc_api" type="ircRobotArmStateMachine" name="ircRobotArmStateMachine" output="screen" />
    </group> -->
</launch>